operations/boot_orchestration/Determine_Which_BOS_Session_Booted_A_Node.md

```diff
@@ -0,0 +1,91 @@
+# Determine Which BOS Session Booted a Node
+
+This guide is split into BOS Version 1 (V1) and BOS Version 2 (V2) sections
+because the procedures are different for the two different versions.
+
+## BOS Version 1 (V1)
+
+To determine which BOS Session booted or rebooted a node, query the node's
+kernel boot parameters. They contain a string 'bos_session_id' which identifies
+which BOS session booted or rebooted the node. Then, use this BOS Session ID
+to describe the BOS Session, which identifies the BOS Session template used.
+
+### Query the Node
+
+(`ncn-mw#`) Query the node in question via ssh. The returned value will be the session ID.
+
+```bash
+BOS_SESSION_ID=$(ssh <node's xname> "cat /proc/cmdline" | awk -v RS=" " -F "=" '{if ($1 == "bos_session_id") { print $2; }}')
+
+```
+
+### Query BOS
+
+(`ncn-mw#`) Ask BOS to describe this session.
+
+'''bash
+cray bos v1 session describe ${BOS_SESSION_ID} --format json
+'''
+
+The templateName parameter is the BOS session template used to boot or reboot the node.
```

Comment:
````suggestion
This will output information about the BOS session. For example:

```
{
  "complete": false,
  "error_count": 0,
  "in_progress": false,
  "job": "boa-147b09de-59a8-4444-9bcb-9b54ac7d78cc",
  "operation": "reboot",
  "start_time": "2023-06-23T20:57:34.352623Z",
  "status_link": "/v1/session/147b09de-59a8-4444-9bcb-9b54ac7d78cc/status",
  "stop_time": "2023-06-23 21:24:14.647779",
  "templateName": "knn-boot-x3000c0s28b4n0"
}
```

The `templateName` parameter is the name of the BOS session template used to boot or reboot the node.
````